docs/source/site.md

```diff
@@ -0,0 +1,23 @@
+---
+html_meta:
+  "description": ""
```

Comment:
"description": "Site endpoint for Plone REST API"
  "property=og:description": "Site endpoint for Plone REST API"